package.json

```diff
@@ -47,26 +47,34 @@
   },
   "homepage": "https://github.com/jeffijoe/typesync#readme",
   "dependencies": {
+    "ansis": "^3.17.0",
     "awilix": "^12.0.5",
-    "chalk": "^5.4.1",
-    "cosmiconfig": "^9.0.0",
     "detect-indent": "^7.0.1",
-    "js-yaml": "^4.1.0",
+    "lilconfig": "^3.1.3",
     "npm-registry-fetch": "^18.0.2",
-    "ora": "^8.1.1",
+    "picospinner": "^3.0.0",
     "semver": "^7.7.0",
-    "tinyglobby": "^0.2.12"
+    "tinyglobby": "^0.2.12",
+    "yaml": "^2.7.1"
+  },
+  "peerDependencies": {
+    "jiti": "^2.4.2"
```

Comment:
Jiti is a large dependency, so I made it optional on the assumption that if you care enough about ts to write a config in ts, you've probably also got Jiti installed already.

Reply:
Never heard of Jiti - is it worth adding the extra dependency?

Reply:
As I said, it's optional so I don't think there's really a cost to it.